simple_history/models.py

```diff
@@ -74,15 +83,11 @@ def save_without_historical_record(self, *args, **kwargs):
 
     def finalize(self, sender, **kwargs):
         inherited = False
-        try:
-            hint_class = self.cls
-        except AttributeError:  # called via `register`
-            pass
-        else:
-            if hint_class is not sender:  # set in concrete
-                inherited = (self.inherit and issubclass(sender, hint_class))
-                if not inherited:
-                    return  # set in abstract
+        if self.cls is not sender:  # set in concrete
```

Comment:
Will this still work when you call register?

Reply:
Yes I updated the register method to store this and the unit tests have been updated to verify it.

Reply:
What about `populate_history`? I noticed this removed adding the model to `registered_models` which is used there

Reply:
The register function calls calls `records.finalize(model)` on line 34.  The `finalize` method calls `create_history_model` which also updates `registered_models` on line 148.